enterprise_gateway/tests/test_handlers.py

```diff
@@ -395,7 +395,7 @@ def test_kernel_comm(self):
         }))
 
         # Assert the reply comes back. Test will timeout if this hangs.
-        for _ in range(3):
+        for _ in range(4):
```

Comment:
If you force Notebook to be 6.1.x other then 6.1.6, would this still work?

Reply:
Yes.  Prior versions will terminate the loop on the third iteration (rather than the fourth).